src/PostgREST/App.hs

```diff
@@ -206,8 +204,13 @@ app dbStructure conf apiRequest =
         else return notFound
 
     (ActionRead, TargetRoot, Nothing) -> do
-      body <- encode <$> H.query schema accessibleTables
-      return $ responseLBS status200 [jsonH] $ cs body
+      let encodeApi ti = encodeOpenAPI ti host port
```

Comment:
Just realised that `X-Forwarded-URI` will help also with the generation of the Location header so maybe leave that for another PR and let's merge this one as is.

The next PR will address swagger/location behind a proxy separatly